src/widgets/splits/Split.cpp

```diff
@@ -298,6 +303,19 @@ void Split::onAccountSelected()
     }
 
     this->input_->ui_.textEdit->setPlaceholderText(placeholderText);
+
+    this->updateTooltipColor();
+    this->signalHolder_.managedConnect(this->theme->updated, [this]() {
+        this->updateTooltipColor();  //
```

Comment:
```suggestion
        this->updateTooltipColor();
```

Reply:
Single line lambxas will be disabled soon tm so the code is good as is